test/doc_comments_test.dart

```diff
@@ -0,0 +1,199 @@
+// Copyright 2018 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'package:sass/src/ast/sass.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('documentation comments', () {
+    group('SCSS syntax:', () {
+      test('variable declarations are found', () {
```

Comment:
I usually try to name my tests and groups so that they (more or less) form full sentences when combined. So I might write this:

```dart
group('documentation comments', () {
  group('in SCSS', () {
    group('attach to variable declarations', () {
```

Reply:
Done.